static/app/utils/useKeyPress.tsx

```diff
@@ -11,6 +11,7 @@ const useKeyPress = (targetKey: string, target?: HTMLElement) => {
     const downHandler = ({key}: KeyboardEvent) => {
       if (key === targetKey) {
         setKeyPressed(true);
+        setTimeout(() => setKeyPressed(false), 200);
```

Comment:
This kinda defeats the purpose of this hook also and likely opens it up to bugs.

For example, if I was using this hook to display something while a key is pressed, it would only display for 200ms before dissapearing.

Reply:
🤔 that's true

I wonder if that use case would necessitate a separate useKeyHold method though?

since there's already an existing bug though, I'd argue this is trading a current real bug for a theoretical bug